tests/sdt-test.py

```diff
@@ -0,0 +1,450 @@
+#! /usr/bin/python3
+
+# Copyright (C) 2017 - Erica Bugden <erica.bugden@efficios.com>
+#                      Francis Deslauriers <francis.deslauriers@efficios.com>
+#
+# This library is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; version 2.1 of the License.
+#
+# This library is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301 USA
+
+import sys
+import subprocess
+import os.path
+import collections
+import re
+import binascii
+from ctypes import *
+
```

Comment:
Here, explain our definition of offset and address and why we need this function.